lib/http/request.js

```diff
@@ -189,18 +189,16 @@ const create = async function (method, path, options, hooks = {}) {
     return options.pathParams[args[1]] || args[0]
   })
 
-  const hasQueryParams = options.queryParams ? Object.keys(options.queryParams).length > 0 : false
```

Comment:
Don't need this anymore now that we can depend on `options.queryParams` either existing if it contains something and not existing if there are no query params passed as options.